.github/workflows/test-job.yaml

```diff
@@ -88,12 +88,92 @@ jobs:
         $(go env GOPATH)/bin/goimports -d apiserver/ ray-operator/ cli/
       if: failure()
 
-  build:
+  build_apiserver:
     env:
-      working-directory: ./ray-operator
+      working-directory: ./apiserver
       cli-working-directory: ./cli
-      apiserver-working-directory: ./apiserver
-    name: Build Binaries and Docker Images
+    name: Build Apiserver, CLI Binaries and Docker Images
```

Comment:
trying to understand the motivation here? Anything wrong with the previous setting?

Reply:
This can make the github pipeline finish faster.